framework/assets/yii.js

```diff
@@ -144,8 +144,8 @@ yii = (function ($) {
          * @param $e the jQuery representation of the element
          */
         handleAction: function ($e, event) {
-            var method = $e.data('method'),
-                $form = $e.closest('form'),
+            var $form = $e.attr('data-form') ? $('#' + $e.attr('data-form')) : $e.closest('form'),
```

Comment:
@SamMousa I accept your cons of using `$.data()` method. But I like `$.data()` because it's shorter and more clear. There is also a good answer on SO with detailed explanation: http://stackoverflow.com/questions/7261619/jquery-data-vs-attr?answertab=votes#tab-top

@yiisoft/core-developers since it's a disputable question, I would like to hear your thoughts